taxonomies/ingredients_processing.txt

```diff
@@ -1135,3 +1135,13 @@ de:aus Früchten
 
 #to be analysed:
 #fr:rectifié
+
+#######################################################################
+#
+#                U L T R A - P R O C E S S I N G   P R O C E S S E S
+#
+#######################################################################
+
+en:hydrolysed
```

Comment:
can we do that in a separate PR?

we would need corresponding code changes so that NOVA 4 ingredient processing are used when computing NOVA groups.

Reply:
I'll comment it out for the moment.